README.md

```diff
@@ -1,33 +1,45 @@
-Recommended node version: `v12.3.0`
+# Envelop
 
-Install dependencies:
-```
-yarn install
-```
+## Contribute
+
+### Setup
 
-Run docker-compose:
+- Make sure `node` is available. Recommended version is `v12.3.0`.
+- Make sure `docker` and `docker-compose` is available.
+
+Run containers
 ```
 docker-compose up
 ```
 
-Start local server with access to production data:
-(available at http://localhost:3000, ignore netlify's cli output)
+Install dependencies:
 ```
-npm run netlify-local
+yarn install
 ```
 
-Start live demo server with access to production data:
+Start local server:
+(available at http://localhost:3000, ignore netlify's cli output)
 ```
-npm run netlify-live
+npm run netlify-local
 ```
 
 Open in  browser:
 ```
-http://localhost:8888
+http://localhost:3000
 ```
 
-Test with sample file (the following is just an example, it doesn't necessarily
-work):
-```
-http://localhost:3000/d/joaodiogocosta.id.blockstack/hello
-```
+### Guide
+
+1. Fork it ( https://github.com/envelop-app/envelop-web )
```

Comment:
You usually use back-ticks for markdown inlined code.

Reply:
@sdsantos yup I'm aware... But where are you suggesting back-ticks? For the `git checkout ...` stuff?

Reply:
If that's what you're talking about, done 👍